Source/Common/IGListDiff.h

```diff
@@ -30,6 +30,17 @@ typedef NS_ENUM(NSInteger, IGListDiffOption) {
 };
 
 /**
+ Comparison behavior options.
+ */
+typedef NS_OPTIONS(NSInteger, IGListDiffBehavior) {
+    IGListDiffBehaviorDefault = 0,
```

Comment:
@jessesquires I don’t see any differences in the generated Swift interface whether the option is named `default` or `none`.

[Documentation](https://developer.apple.com/library/content/documentation/Swift/Conceptual/BuildingCocoaApps/InteractingWithCAPIs.html#//apple_ref/doc/uid/TP40014216-CH8-NoLink_29) never mentions `none` either:
> When importing C enumeration marked with the NS_OPTIONS macro, Swift marks any members that have a value of 0 as unavailable, because Swift uses an empty option set to specify no options.